### ISSUE TITLE
Reply list wording revamp

### ISSUE THREAD

**Issue by MrOrz** [cofacts/rumors-line-bot#140]
From: https://g0v.hackmd.io/CWKXj50SRQuCfxxlFi3STg#2-%E6%9C%89%E5%A4%9A%E5%89%87%E5%9B%9E%E6%87%89%E6%99%82

Needs clearer call-to-action so that users know that they should choose.

![](https://g0vhackmd.blob.core.windows.net/g0v-hackmd-images/upload_09a15b9d1143226aefe63fe9ee532925)

```
Cofacts 志工對這則訊息發表了多則看法唷！

👨‍👩‍👧‍👦有 A 人說它 ❌ 含有不實訊息、B 人說它 ⭕ 含有真實訊息。

選一則來閱讀吧👇
```

**Comment by MrOrz** [comment]
```
{
  "type": "carousel",
  "contents": [
    {
      "type": "bubble",
      "direction": "ltr",
      "body": {
        "type": "box",
        "layout": "vertical",
        "contents": [
          {
            "type": "text",
            "text": "💬 有志工認為 ❌  含有不實訊息",
            "size": "sm"
          },
          {
            "type": "text",
            "text": "假的。恐怖份子或是培訓示威都是無稽之談。奧斯陸自由論壇、 （英語：Oslo Freedom Forum）是在挪威政府支持下由位於紐約的美國人權基金會於2009年創辦，每年在諾貝爾和平獎主辦城市奧斯陸舉辦，奧斯陸自由論壇的與談人包括各國前總統、諾貝爾獎得主、良心犯與其他公眾人物，主⋯⋯",
            "align": "start",
            "wrap": true,
            "margin": "md",
            "maxLines": 10
          },
          {
            "type": "separator",
            "margin": "md"
          },
          {
            "type": "box",
            "layout": "horizontal",
            "contents": [
              {
                "type": "text",
                "text": "👍 5 人覺得有幫助",
                "size": "xs",
                "wrap": true
              },
              {
                "type": "text",
                "text": "😕 3 人覺得沒幫助",
                "size": "xs",
                "wrap": true
              }
            ],
            "margin": "md",
            "spacing": "none"
          }
        ]
      },
      "footer": {
        "type": "box",
        "layout": "vertical",
        "contents": [
          {
            "type": "button",
            "action": {
              "type": "uri",
              "label": "👀 看他怎麼說",
              "uri": "https://linecorp.com"
            },
            "style": "primary"
          }
        ]
      }
    },
    {
      "type": "bubble",
      "direction": "ltr",
      "body": {
        "type": "box",
        "layout": "vertical",
        "contents": [
          {
            "type": "text",
            "text": "💬 也有志工認為 ❌  含有不實訊息",
            "size": "sm"
          },
          {
            "type": "text",
            "text": "假的。恐怖份子或是培訓示威都是無稽之談。奧斯陸自由論壇、 （英語：Oslo Freedom Forum）是在挪威政府支持下由位於紐約的美國人權基金會於2009年創辦，每年在諾貝爾和平獎主辦城市奧斯陸舉辦，奧斯陸自由論壇的與談人包括各國前總統、諾貝爾獎得主、良心犯與其他公眾人物，主⋯⋯",
            "align": "start",
            "wrap": true,
            "margin": "md",
            "maxLines": 10
          },
          {
            "type": "separator",
            "margin": "md"
          },
          {
            "type": "box",
            "layout": "horizontal",
            "contents": [
              {
                "type": "text",
                "text": "👍 555 人覺得有幫助",
                "size": "xs",
                "wrap": true
              },
              {
                "type": "text",
                "text": "😕 777 人覺得沒幫助",
                "size": "xs",
                "wrap": true
              }
            ],
            "margin": "md",
            "spacing": "none"
          }
        ]
      },
      "footer": {
        "type": "box",
        "layout": "vertical",
        "contents": [
          {
            "type": "button",
            "action": {
              "type": "uri",
              "label": "👀 看他怎麼說",
              "uri": "https://linecorp.com"
            },
            "style": "primary"
          }
        ]
      }
    }
  ]
}
```

**Comment by MrOrz** [comment]
Implemented in #146
